pkg/acceptance/helpers/database_client.go

```diff
@@ -25,6 +25,30 @@ func (c *DatabaseClient) client() sdk.Databases {
 	return c.context.client.Databases
 }
 
+func (c *DatabaseClient) CreatePrimaryDatabase(t *testing.T, enableReplicationTo []sdk.AccountIdentifier) (*sdk.Database, sdk.ExternalObjectIdentifier, func()) {
```

Comment:
nit-pick: let's not make it the first method in this file (look where secondary is located)

Reply:
Moved it under secondary.